applications/MultilevelMonteCarloLaplacianApplication/python_scripts/pure_diffusion_solver.py

```diff
@@ -0,0 +1,241 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```

Comment:
how is this different from the solver in the ConvectionDifusionApplication?

Reply:
Actually, the problem solved should be a black box, meaning that the MultilevelMontecarloApplication shouldn't implement any physics. Isn't it?

Reply:
Yes @rubenzorrilla  you are right, but here I implemented also the solver. As @RiccardoRossi  said in a comment, I should use the convection diffuzion solver, if possible.